### DISCUSSION TITLE
Podman 4.3 RPM

### DISCUSSION THREAD

**Discussion by talha110** [containers/podman#16298] [Q&A] [answered]
Hi folks, Is there any way I can install Podman 4.3 in Rhel 8.6? 
There are no RPMs for CentOS8 yet
http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/
https://rpmfind.net/linux/rpm2html/search.php?query=podman
Thank you

**Comment by vrothberg** [comment, marked as answer]
Thanks for reaching out, @talha110!

> Hi folks, Is there any way I can install Podman 4.3 in Rhel 8.6?

No, Podman v4.3 is not scheduled for RHEL, so it won't be available in CentOS.  Podman v4.2 is in RHEL 8.7.  Podman v4.4 (or later) will make it into RHEL 8.8.

**Reply by talha110** [reply]
Thanks for the reply @vrothberg, 
Well, that's a bummer, I was really looking forward to using 4.3 since it has the network recreation fix merged (https://github.com/containers/podman/issues/15580). Guess I will have to find a workaround for this

**Reply by vrothberg** [reply]
If you're a RHEL customer, please go through the Red Hat customer channels.  There is nothing we can do upstream but the customer channels can help you.

**Comment by rhatdan** [comment]
RHEL 8.7.0.2 will have podman v4.3 or V4.4 correct?

**Reply by mheon** [reply]
Should have 4.4 AFAIK
